cmd/statusd/main.go

```diff
@@ -54,6 +54,7 @@ var (
 	pprofPort         = flag.Int("pprofport", 52525, "Port for runtime profiling via pprof")
 	logLevel          = flag.String("log", "INFO", `Log level, one of: "ERROR", "WARN", "INFO", "DEBUG", and "TRACE"`)
 	logFile           = flag.String("logfile", "", "Path to the log file")
+	disableLogColors  = flag.Bool("disable-log-colors", false, "Disables log colors")
```

Comment:
To keep it close to other log-related flags, it should start with `log`. For example, `log-without-colors` or `log-enable-colors` with `true` as a default.

Reply:
I'm better going for `log-without-colors` as enabled colors is the default atm